include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
Since the use of sizeof() will be consistent internally for a given compiler, I think it is safe to say that the only problem is the assumption that a size of one means 8 bits (vs actually 16 bits) when doing bit comparison and display.

As before, I am not sure this needs to be fixed. Version 3.7.2 of CppUTest is still fine with the TI. Nobody has complained so far.